setup.cfg

```diff
@@ -18,17 +18,17 @@ keywords =
 classifiers =
     Development Status :: 4 - Beta
     License :: OSI Approved :: Apache Software License
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
If we're adding 3.10 here, we should add it to CI as well?

Reply:
I'm going to remove 3.10. It "ought to" work, but there are no rasterio wheels for 3.10 yet and that kinda blocks testing and installation for most users.